contracts/CommunityResolver.sol

```diff
@@ -52,10 +52,13 @@ contract CommunityResolver is
      * This is an bottom up event, called from the attest contract
      */
     function onAttest(
-        Attestation calldata attestation,
+        Attestation calldata /*attestation*/,
```

Comment:
This resolver comes into play only when we create a community which is not too often. Let's not change this. No need to handle in our indexer either.